docs/references/config/node_key.json.md

```diff
@@ -0,0 +1,106 @@
+---
+order: 1
+parent:
+  title: node_key.json
+  description: Description and usage of the node ID
+  order: 2
+---
+
+## The `node_key.json` file
+The node ID, the IP address of the host and the P2P port together identify a node: `nodeID@IP:port`.
```

Comment:
We'll have to clarify that better. Although the config can accept DNS names, the address book stores IP addresses. So if you have a round robin DNS name, only the first IP address of it will end up in the address book by default.

Reply:
this information would not be pertinent here because there's not host address configuration here. That blurb just explains how a node is identified. I will use Daniel suggestion and re-word things. If we need to make some clarifications that should be in a section pertinent to that.